server/service/src/sync/translations/invoice_line.rs

```diff
@@ -322,14 +329,32 @@ impl SyncTranslation for InvoiceLineTranslation {
                     number_of_packs,
                     note,
                     inventory_adjustment_reason_id,
-                    return_reason_id: _, // TODO
+                    return_reason_id,
                     foreign_currency_price_before_tax,
                     item_variant_id,
                 },
             item_row,
             ..
         } = invoice_line;
 
+        let invoice = match InvoiceRowRepository::new(connection).find_one_by_id(&invoice_id)? {
```

Comment:
You can just destructure invoice_row off of invoice_line above rather than extra query here 😁

Reply:
Oh you're so right :rofl